pim-tool-back/src/main/java/vn/elca/training/model/entity/Group1.java

```diff
@@ -0,0 +1,51 @@
+package vn.elca.training.model.entity;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
```

Comment:
[N] `@Entity` accepts a value, use this value to rename or escape the table name.